### DISCUSSION TITLE
SUPERSET_ENV Unused?

### DISCUSSION THREAD

**Discussion by bluepython508** [apache/superset#15495] [Q&A / Help]
Does the environment variable `SUPERSET_ENV` have any effect? I can find exactly 3 references to it - [line 76 of `docker/README.md`](https://github.com/apache/superset/tree/master/docker#production), [line 43 of `docker/.env`](https://github.com/apache/superset/tree/master/docker/.env#L43), and [line 43 of `docker/.env-non-dev`](https://github.com/apache/superset/tree/master/docker/.env-non-dev#L43).

**Comment by nytai** [comment]
I don't think it currently changes any behavior in the app.

**Reply by bluepython508** [reply]
Then should it be set at all?

**Reply by nytai** [reply]
It may be used by more customized versions or superset or some future version

**Reply by 1c7** [reply]
## Same question

as of `2022-Feb-7`, I seach for "SUPERSET_ENV" in VSCode, not much showing up

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/1804755/152785699-c0073c83-b2cc-4e2a-bf89-c176cb4d6cfd.png">

## There are no code using `SUPERSET_ENV` at all

**Reply by 1c7** [reply]
## FLASK_ENV on the other hand are clearly in used
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/1804755/152785876-fe869f7f-8026-45f2-bef2-2718f52126d9.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/1804755/152786047-db2efa42-2e94-45ee-a2f6-93f2851337ae.png">

**Reply by 1c7** [reply]
I think we should completely delete `SUPERSET_ENV`

**Reply by 1c7** [reply]
## Question: Should I send a Pull Request in which delete all `SUPERSET_ENV`?
